__tests__/cli.js

```diff
@@ -10,27 +10,27 @@ const cliFilePath = path.join(__dirname, '../cli.js');
 const yarnLockFilePath = path.join(__dirname, '../__fixtures__/yarn.lock');
 
 test('prints duplicates', async () => {
-  const { stdout, stderr } = await exec(`${cliFilePath} --list ${yarnLockFilePath}`);
+  const { stdout, stderr } = await exec(`node ${cliFilePath} --list ${yarnLockFilePath}`);
```

Comment:
Thanks for fixing this.

Nitpick: you can't assume that global `node` is the right `node` binary to use to run the scripts. The best assumption is to reuse the node process that was used to execute this test, which is in `process.execPath`.

Reply:
Good call. Done in 3f2cc964afeb35b74d082891727b9fd0913ae27a.
I decided to replace `exec` with `execFile` as well since I ran into this error when I just replaced `node` with `${process.execPath}`
"'C:\Program' is not recognized as an internal or external command, operable program or batch file."